test/e2e/config/config.go

```diff
@@ -4,12 +4,16 @@ import "github.com/kelseyhightower/envconfig"
 
 // Config holds global test configuration translated from environment variables
 type Config struct {
-	SubscriptionID        string `envconfig:"SUBSCRIPTION_ID"`
-	ResourceGroup         string `envconfig:"RESOURCE_GROUP"`
-	AzureClientID         string `envconfig:"AZURE_CLIENT_ID"`
-	KeyvaultName          string `envconfig:"KEYVAULT_NAME"`
-	KeyvaultSecretName    string `envconfig:"KEYVAULT_SECRET_NAME"`
-	KeyvaultSecretVersion string `envconfig:"KEYVAULT_SECRET_VERSION"`
+	SubscriptionID           string `envconfig:"SUBSCRIPTION_ID"`
+	ResourceGroup            string `envconfig:"RESOURCE_GROUP"`
+	AzureClientID            string `envconfig:"AZURE_CLIENT_ID"`
+	KeyvaultName             string `envconfig:"KEYVAULT_NAME"`
+	KeyvaultSecretName       string `envconfig:"KEYVAULT_SECRET_NAME"`
+	KeyvaultSecretVersion    string `envconfig:"KEYVAULT_SECRET_VERSION"`
+	MICVersion               string `envconfig:"MIC_VERSION"`
+	NMIVersion               string `envconfig:"NMI_VERSION"`
+	Registry                 string `envconfig:"REGISTRY"`
```

Comment:
Is this the variable - REGISTRY,  which will change where the image would get pulled from ?

Reply:
correct. so for prod it would be `mcr.microsoft.com/k8s/aad-pod-identity/`